src/promptflow/tests/sdk_cli_azure_test/e2etests/test_run_operations.py

```diff
@@ -911,3 +912,84 @@ def test_requirements_in_additional_includes(self, pf: PFClient, randstr: Callab
         with TemporaryDirectory() as temp:
             pf.runs.download(run=run.name, output=temp)
             assert Path(temp, run.name, "snapshot/requirements").exists()
+
+    @pytest.mark.skipif(
+        condition=is_live(),
+        reason="removed requirement.txt to avoid compliance check.",
+    )
+    def test_eager_flow_crud(self, pf: PFClient, randstr: Callable[[str], str]):
+        run = pf.run(
+            flow=f"{EAGER_FLOWS_DIR}/simple_with_req",
+            data=f"{DATAS_DIR}/simple_eager_flow_data.jsonl",
```

Comment:
Do you think it's necessary to test column mapping functionality also?

Reply:
This PR is only for basic CRUD operations, let me add more complex cases in the future.